### PR TITLE
During startup, enable text field, but disable submit

### PR DESCRIPTION
**Pull request by 0xSaurabhx**
The disabled attribute is removed from the input element. This allows users to type their prompt even while OpenDevin is initializing.
The disabled attribute is added to the button element and set to !initialized. This ensures that the user cannot submit the prompt until OpenDevin is ready.

### CODE REVIEW

File: frontend/src/components/ChatInterface.tsx [hunk 1]
```diff
@@ -52,23 +52,25 @@ function ChatInterface(): JSX.Element {
...
-              handleSendMessage();
-            }
-          }}
-          disabled={!initialized}
-        />
-        <button type="button" onClick={handleSendMessage}>
-          <span className="button-text">Send</span>
-        </button>
-      </div>
+<div className="input-container">
```
Comment (rbren) [inline frontend/src/components/ChatInterface.tsx:55]: looks like we lost some indent here

**Comment by rbren** [comment]
@0xSaurabhx looks like a lint issue. Otherwise LGTM!

**Comment by 0xSaurabhx** [comment]
Looks like You have already added changes main branch

**Comment by rbren** [comment]
Not sure what happened here--looks like the changed files are unrelated?

The original problem is still there: https://github.com/OpenDevin/OpenDevin/blob/95c128c3d69a4cb3c8b13d1d4aadcbc44d8fdff5/frontend/src/components/ChatInterface.tsx#L66

**Comment by rbren** [comment]
@0xSaurabhx this branch now contains changes unrelated to the PR above. I'm going to close it, but feel free to open a new PR with the proposed changes.

In the future, I'd suggest creating a new branch with `git checkout -b` every time you want to work on a new feature--it keeps the PRs from getting cluttered with unrelated changes

**Pull request closed without merging**